### DISCUSSION TITLE
I'm a Brazilian newbie and I need help with some doubts. Can anyone help me out? Thanks in advance!

### DISCUSSION THREAD

**Discussion by FranciscoNetoBr** [ant-media/Ant-Media-Server#4903] [Q&A] [answered]
Hey guys, I'm new here, and I'm also getting to know the Ant Media Server tool right now.

The use I want to apply is 24/7 streaming on a YouTube channel.

And I need to understand a little bit.

Questions:
Today I have to join audio and video to upload everything in a single .MP4 file.

Is it possible to transmit video and audio separately?
To update videos in the broadcast, do I need to stop the broadcast, include a new playlist, and start the broadcast again?
Is it possible to add new videos or schedule the entry of new videos without stopping the broadcast?
If I can get a positive response, it will help me a lot, especially the possibility of scheduling the entry of a video at a certain time, without having to stop the broadcast and doing it automatically.

Thanks in advance for your support in clarifying this doubt for me.

**Comment by yashtandon113** [comment, marked as answer]
Hi @FranciscoNetoBr 
Thanks for asking these good questions. Let me try to answer.

1. You can use Ant Media Server to create separate playlists with audio and video only files, but you can't publish audio and video only files to YouTube because it doesn't accept streams that don't include video or audio. A stream should include both audio and video, according to the guidelines. Even if no audio is present, a dummy audio track should be included.

For example, when publishing a video-only stream, OBS studio, in my observatio, adds a dummy audio track. AMS does not add any dummy tracks by default. To create a dummy audio or video track in AMS, use the filter plugin, which is available in the Enterprise edition.
 
Please check the following post for reference: https://community.antmedia.io/topic/1418-cant-get-to-stream-to-youtube-from-ant-media-app-community-edition-in-linode/?do=findComment&comment=5216

2. No, you are not required to pause the broadcast. The [update broadcast](https://antmedia.io/rest/#/BroadcastRestService/updateBroadcast) Rest API call can be used to add the new video to the playlist. Please see the Postman CURL sample below to update the Playlist.

In this example, the test1.mp4 file was already added to the playlist, and the update broadcast was used to add the test2.mp4 file without interrupting the stream.

```
curl --location --request PUT 'http://localhost:5080/LiveApp/rest/v2/broadcasts/test' \
--header 'Content-Type: application/json' \
--data '{"playListItemList": [
        {
            "streamUrl": "http://localhost:5080/LiveApp/streams/test1.mp4",
            "type": "VoD"
        },
        {
            "streamUrl": "http://localhost:5080/LiveApp/streams/test2.mp4",
            "type": "VoD"
        }
    ]}'
```

3. As I mentioned in point 2, you do not need to stop the broadcast because the newly added video to the playlist will play in sequence, but Ant Media Server does not have a feature that allows you to schedule the start/stop playlist at any time.

You can, however, use the [start](https://antmedia.io/rest/#/BroadcastRestService/startStreamSourceV2) and [stop](https://antmedia.io/rest/#/BroadcastRestService/stopStreamingV2) rest APIs and write logic to call these at a specific date or time.

You can also open a feature-request issue, and we will include it in the backlog.
https://github.com/ant-media/Ant-Media-Server/issues


I hope it helps.

**Reply by FranciscoNetoBr** [reply]
I want to thank you for your time. I took a while to respond because I was assimilating the answer. The observations and alternatives provided are not really "simple" and intuitive. However, I will work on finding the best way to address my issue.
